unit-tests/func/func-common.h

```diff
@@ -94,3 +135,30 @@ inline stream_profile find_confidence_corresponding_to_depth( rs2::depth_sensor
     REQUIRE( confidence_profile != stream_profiles.end() );
     return *confidence_profile;
 }
+
+
+bool get_profile_by_stream_parameters( rs2::depth_sensor ds,
+                                       rs2_stream stream,
+                                       rs2_format format,
+                                       int width,
+                                       int height,
+                                       rs2::stream_profile & required_profile )
```

Comment:
Why not just return a `stream_profile`?

Reply:
I prefer that functions have a return value, instead of having to wrap the output with try...catch

What is it didn't find a matching profile?

What do you think?